src/parameters.ml

```diff
@@ -39,21 +39,21 @@ let initial_parameters : parameters =
   }
 
 (** Compute the current minting index (in tez). To get tez/kit must multiply with q. *)
-let[@inline] tz_minting (p: parameters) : Ligo.tez = max_tez p.index p.protected_index
+let[@inline] tz_minting (p: parameters) : Ligo.nat = max_nat p.index p.protected_index
 
 (** Compute the current liquidation index (in tez). To get tez/kit must multiply with q. *)
-let[@inline] tz_liquidation (p: parameters) : Ligo.tez = min_tez p.index p.protected_index
+let[@inline] tz_liquidation (p: parameters) : Ligo.nat = min_nat p.index p.protected_index
 
 (** Current minting price (in tez/kit). *)
 let minting_price (p: parameters) : ratio =
   make_ratio
-    (Ligo.mul_int_int (fixedpoint_to_raw p.q) (tez_to_mutez (tz_minting p)))
+    (Ligo.mul_int_int (fixedpoint_to_raw p.q) (Ligo.int (tz_minting p)))
     (Ligo.mul_int_int (fixedpoint_to_raw fixedpoint_one) (Ligo.int_from_literal "1_000_000"))
 
 (** Current liquidation price (in tez/kit). *)
 let liquidation_price (p: parameters) : ratio =
   make_ratio
-    (Ligo.mul_int_int (fixedpoint_to_raw p.q) (tez_to_mutez (tz_liquidation p)))
+    (Ligo.mul_int_int (fixedpoint_to_raw p.q) (Ligo.int (tz_liquidation p)))
```

Comment:
It's nice that we don't have to do the extra conversion from tez to int here :+1:

Reply:
Yeah, that's nice indeed! I expect other places can be simplified too.